src/System.Runtime.Numerics/src/System/Numerics/BigCalc.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace System.Numerics
+{
```

Comment:
Actually, I'm guessing that you will want to have other partial files for additional operations in the future. I think we can still probably get rid of this empty file in that case.